### DISCUSSION TITLE
Secondary colony

### DISCUSSION THREAD

**Discussion by ghost** [ldtteam/minecolonies#10307] [Ideas]
This add-on should be added if you added the expedition function: Add the function of creating a secondary colony (this is like a continuation of the colony, but in a different place). How it works: when colonists come from an expedition, they can say "During the journey, we found an abandoned colony at coordinates X Y Z, would you like to capture it?" And after that, you could restore this colony and it would be yours, you could send residents and resources from one colony to another. For example, you can arrange this with a donkey or a mule, on which you can hang a chest (for transporting resources), You can also add that residents learn to swim on boats and boats with chests.

**Comment by Ivan228078ebalo** [comment]
Святую Кебабию ждёт холокост
